Changelog.md

```diff
@@ -2,6 +2,68 @@
 
 ### libsdformat 9.X.X (202X-XX-XX)
 
+### libsdformat 9.8.0 (2022-04-26)
+
+1. Polyline geometry DOM
+    * [Pull request #1000](https://github.com/ignitionrobotics/sdformat/pull/1000)
+
+1. Added <shininess> to <material>
```

Comment:
```suggestion
1. Added `<shininess>` to `<material>`
```

backticks are needed for `<.*>` to render properly (preview the file in GitHub to confirm)